### DISCUSSION TITLE
在Termux上尝试使用yarn安装时提示平台“android”与此模块不兼容。

### DISCUSSION THREAD

**Discussion by Akilarlxh** [koishijs/koishi#345] [Q&A]
**Is your feature request related to a problem? Please describe.**

在使用termux在安卓手机上进行使用的时候，提示报错
error systeminformation@5.7.8: The platform "android" is incompatible with this module.

**Describe the solution you'd like**

有没有可能使其兼容安卓平台。或者说是兼容arm。目前只能在手机上运行go-cqhttp，却不能使用koishi，乐趣一下少了一大半。
**Describe alternatives you've considered**

**Additional context**

**Comment by NWYLZW** [comment]
搞点详细报错看看，nodejs 能跑应该就没问题

**Comment by NWYLZW** [comment]
![Screenshot_2021-07-27-14-32-24-205_com.termux.jpg](https://user-images.githubusercontent.com/51358815/127107059-747aa2fe-0cdf-4ed5-88e6-9c59156a8045.jpg)
![Screenshot_2021-07-27-14-33-33-826_com.termux.jpg](https://user-images.githubusercontent.com/51358815/127107066-6e261fd5-d66b-4d49-9f78-545d3a484331.jpg)

我没有任何问题

**Comment by shigma** [comment]
@NWYLZW 这位的报错好像是跟 plugin-webui 有关的，不是单纯的 koishi

**Comment by shigma** [comment]
@Akilarlxh 贴下你的 package.json，你大概是用了不支持 Android 平台的插件导致，并不是 koishi 本身无法运行

**Comment by NWYLZW** [comment]
> @NWYLZW 这位的报错好像是跟 plugin-webui 有关的，不是单纯的 koishi

应该不是 koishi 的问题吧
不过 webui 确实要去获取一些系统数据
我觉得可能是 termux 没给 api 或者他没给 termux 这方面的权限
